src/cljs/rems/actions/action.cljs

```diff
@@ -134,3 +135,14 @@
                       (collapse-action-form collapse)
                       (on-finished)))
           :error-handler (flash-message/default-error-handler :actions description)}))
+
+(defn guide []
+  [:div
```

Comment:
How about `component-info` here?

Reply:
I thought it's not that important since `action-attachment-view` isn't used anywhere, the `action-attachment` wrapper is